rxnetty-http/src/test/java/io/reactivex/netty/protocol/http/client/HttpClientTest.java

```diff
@@ -272,6 +274,101 @@ public void call() {
 
     }
 
+    @Test(timeout = 60000)
+    public void testRequestWithNoContentLengthHeaderOrContentReturnsEmptyBody() {
+        serverRule.startServer();
+
+        serverRule.assertRequestEquals(
+            new Func1<HttpClient<ByteBuf, ByteBuf>, Observable<HttpClientResponse<ByteBuf>>>() {
+                @Override
+                public Observable<HttpClientResponse<ByteBuf>> call(HttpClient<ByteBuf, ByteBuf> client) {
+                    return client.createGet("/");
+                }
+            },
+            "GET / HTTP/1.1\r\n" +
```

Comment:
Not sure if checking raw HTTP message is required, can you pass a custom handler that asserts request headers and content inside the server and store the error/success to be queried later?